UnityProject/Assets/Scripts/InGameEvents/InGameEventScripts/EventAsteroids.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace InGameEvents
+{
+	public class EventAsteroids : EventScriptBase
+	{
+		private MatrixInfo stationMatrix;
+
+		private Queue<Vector3> impactCoords = new Queue<Vector3>();
+
+		public GameObject ExplosionPrefab = null;
```

Comment:
Public is not necessary here. Use private with [SerializeField] attribute instead.

Reply:
Done